wormhole_attester/sdk/js/src/index.ts

```diff
@@ -38,13 +22,149 @@ export type BatchPriceAttestation = {
   priceAttestations: PriceAttestation[];
 };
 
-export async function parseBatchPriceAttestation(
```

Comment:
Name change?

Reply:
Hmm no the signature is the same, only the arg name has changed.